### PR TITLE
Update web-build-tools to build with Rush4

### PR DESCRIPTION
**Pull request by nickpape-msft**
Update Rush to 4.0.0 and add a custom command line for web-build-tools

### CODE REVIEW

File: common/config/rush/command-line.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "customCommands": [
+    {
+      "name": "blah",
+      "description": "Do some really, really cool custom stuff! This is the best command!"
```
Comment (pgonzal) [inline common/config/rush/command-line.json]: >Do some really, really cool custom stuff! This is the best command! [](start = 22, length = 67)

Should we support ts-command-line summary/documentation?

File: common/config/rush/command-line.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
+{
+  "customCommands": [
```
Comment (pgonzal) [inline common/config/rush/command-line.json]: >customCommands [](start = 3, length = 14)

Should there be a switch to say whether the command is parallelized or not?

File: common/config/rush/command-line.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  "customOptions": {
+    "--profile": {
+      "optionType": "enum",
+      "description": "Changes the build profile, affecting the build output",
+      "supportedCommands": [ "build", "rebuild" ],
+      "enumValues": [
+        {
+          "name": "vso",
+          "description": "For running on CI servers"
```
Comment (pgonzal) [inline common/config/rush/command-line.json]: >"description": "For running on CI servers" [](start = 10, length = 42)

How does ts-command-line expose this?

Comment (pgonzal) [reply common/config/rush/command-line.json]: Maybe Rush could just internally make them bullets and append them to the overall description?

---
In reply to: [146730671](https://github.com/Microsoft/web-build-tools/pull/396#discussion_r146730671) [](ancestors = 146730671)

**Comment by iclanton** [comment]
Approved.

**Pull request merged**